flask_security/views.py

```diff
@@ -45,6 +45,8 @@ def _render_json(form, include_user=True, include_auth_token=False):
         response = dict()
         if include_user:
             response['user'] = dict(id=str(form.user.id))
```

Comment:
@marcosschroh can you include your last comment in docs? I think it will be very helpful. Thanks!